sound/soc/intel/common/soc-acpi-intel-cnl-match.c

```diff
@@ -33,6 +33,14 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_cnl_machines[] = {
 		.sof_tplg_filename = "intel/sof-cnl.tplg",
 		.asoc_plat_name = "0000:00:0e.0",
 	},
+	{
+		.id = "10EC5682",
+		.drv_name = "sof_rt5682",
+		.pdata = &cnl_pdata,
```

Comment:
this is only needed for the Skylake driver, but since you haven't specified a firmware for it then it will not even work so this field can be removed.